tests/cstdio/test_remove.cpp

```diff
@@ -57,7 +57,7 @@ void test() {
   err = std::remove("dir");
   assert(!err);
 
-  std::cout << "success";
```

Comment:
Can you submit these two test changes seperately?  I think they are useful on their own.